### PR TITLE
Introduction To Network Management 

### LINKED ISSUE #1304: Introduction to Network Management
### Brief Summary: 
Enterprises must be able to accurately track network performance to ensure users can access the applications they need. Find out about the latest trends shaping network monitoring software and tools, including expert advice about the steps companies should take to best manage their networks.

### Key Takeaways: 
1. Network Resource Monitoring Techniques
2. Simple Network Management Protocol
3. ISO Network Management Model
### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by jumamiller**
### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
This fixes #1304

### CODE REVIEW

**Comment by AakashR2208** [comment]
@hectorkambow, I'll review this.

**Comment by jumamiller** [comment]
@AakashR2208 are you still reviewing this?

**Comment by AakashR2208** [comment]
Yeah! I'll post the edits by EOD today.

File: articles/getting-started-with-network-management/index.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+This provides a very efficient fault resolution, hence restoring or solving problems as soon as they occur.  
+
+
+#### ```2.Security Management```
+
+Your organization where you work is in one way or the other a target by hackers.  
+Security management is therefore concerned with how the ``` encryption keys``` are being generated, distributed, and stored.  
+This may involve passwords, authorization keys, and access control information which are usually distributed and requires maintenance.  
+
+Security management is also concerned with computer network access.  Who is accessing our network? where is he coming from? what is he doing on our network?
```
Comment (AakashR2208) [inline articles/getting-started-with-network-management/index.md:64]: Please refrain from using any pronouns in the articles.

File: articles/getting-started-with-network-management/index.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+Security management is also concerned with computer network access.  Who is accessing our network? where is he coming from? what is he doing on our network?  
+It simply involves monitoring and controlling the network.  
+
+One of the most important tools in security management is ```Network Logs```.
+It records every single activity or event on the network. Storing user details, process calls, and authentication attempts.  
+
+Security will always provide a guaranteed user information and network resources protection.  
+An organization should ensure these security facilities are made available to authorized users only.  
+
+In some situations, users will ask questions, take, for example, e-commerce sites, some will need your credit/debit card details, what are their security policies, and are the security systems secure themselves.
```
Comment (AakashR2208) [inline articles/getting-started-with-network-management/index.md:73]: take, for example is redundant

**Review by AakashR2208 — COMMENTED** [review]
Hi! Great topic for an article. Just a couple of quick notes. 
1. Check the overall grammar of the article. At some places, there are some typos, redundant words and grammatical mistakes. 
2. Use the proper formatting guides of Section. 
3. The anecdotes in the article are great but maybe could have written better to sound more professional. Overall the article lacks that professional read to it. So maybe reword some sentences. 
4. Maybe you should go a little deeper content wise. If the content is too much then maybe cut some things out and have a separate article for it. For eg, for fault management and security management, list some examples and technologies to accomplish the task.

**Comment by jumamiller** [comment]
@AakashR2208 most changes are done,

File: articles/getting-started-with-network-management/index.md [hunk 3]
```diff
@@ -8,13 +8,13 @@ In this tutorial, we're going to discuss the fundamentals and principles of netw
...
-Management simply refers to the set of activities or principles well defined to perform a given task effectively.  
-It involves controlling, organizing, planning, monitoring, controlling resources to achieve the organization's goals.
+Management refers to the set of activities or principles well defined to perform a particular task effectively.  
+It involves controlling, organizing, planning, and monitoring resources to achieve the organization's goals.
 
 Network management, therefore, includes all the mechanisms put in place to monitor, control, and coordinate network activities within an organization.  
 These activities include network protocol analysis, distributed database management, and so many other tasks.  
 
-In essence, network management as a service provides network managers with several applications, tools, and devices to ensure there is smooth network management and monitoring.  
+In essence, network management as a service provides network managers with several applications, tools, and devices to ensure there are smooth network management and monitoring.
```
Comment (AakashR2208) [inline articles/getting-started-with-network-management/index.md:17]:
```suggestion
In essence, network management as a service provides network managers with several applications, tools, and devices to ensure smooth network management and monitoring.  
```

**Review by AakashR2208 — COMMENTED** [review]
All right! The article looks good! Thanks for the edits.

**Comment by jumamiller** [comment]
updated

**Comment by jumamiller** [comment]
@AakashR2208, is it ready for final review?

**Comment by AakashR2208** [comment]
Yes @jumamiller . Thanks for the changes. It's a really nice article. 

Hi @hectorkambow , Can you look at this for a final review?

**Comment by jumamiller** [comment]
@AakashR2208, he hasn't seen this(skipped this), could you notify him on slack?

**Comment by AakashR2208** [comment]
I pinged him on slack.

**Comment by hectorkambow** [comment]
@jumamiller  Please note the changes made during final review. I found the article a bit difficult to understand at times and found myself re-wording parts of it. 

cc @AakashR2208  please do the same to improve future article revisions . 

🚀  

Overall great work gentlemen
